release/pkg/manager/calico/manager.go

```diff
@@ -514,14 +522,14 @@ func (r *CalicoManager) releasePrereqs() error {
 		return fmt.Errorf("cannot cut release from branch: %s", branch)
 	}
 
-	// Make sure we have a github token - needed for publishing to GH.
+	// Make sure we have the right token needed for publishing.
```

Comment:
I don't understand this comment change - seems fine, but the other text was also fine and more specific.